routes/api.php

```diff
@@ -14,6 +14,13 @@
 Route::put('/managers/{id}', 'ManagerController@update')->name('managers.update');
 Route::delete('/managers/{id}', 'ManagerController@destroy')->name('managers.destroy');
 
+Route::post('/managers/register', 'ManagerController@register')->name('managers.register');
+Route::post('/managers/login', 'ManagerController@login')->name('managers.login');
+
+Route::group(['middleware' => ['jwt.verify']], function() {
```

Comment:
I'd replace this with [controller middleware](https://laravel.com/docs/6.x/controllers#controller-middleware).